src/Metadata/Resource/Factory/OperationDefaultsTrait.php

```diff
@@ -87,6 +104,38 @@ private function addDefaultGraphQlOperations(ApiResource $resource): ApiResource
         return $resource->withGraphQlOperations($graphQlOperations);
     }
 
+    /**
+     * Adds nested query operations if there are no existing query ones on the resource.
+     * They are needed when the resource is queried inside a root query, using a relation.
+     * Since the nested argument is used, root queries will not be generated for these operations.
+     */
+    private function completeGraphQlOperations(ApiResource $resource): ApiResource
+    {
+        $graphQlOperations = $resource->getGraphQlOperations();
+
+        $hasQueryOperation = false;
+        $hasQueryCollectionOperation = false;
+        foreach ($graphQlOperations as $operation) {
+            if ($operation instanceof Query && !$operation instanceof QueryCollection) {
+                $hasQueryOperation = true;
+            }
+            if ($operation instanceof QueryCollection) {
+                $hasQueryCollectionOperation = true;
+            }
+        }
+
+        if (!$hasQueryOperation) {
+            $queryOperation = (new Query())->withNested(true);
```

Comment:
mb comment on Nested or here that this operation is then use when we fetch a query through another query

Reply:
Isn't the comment on the method sufficient?